lib/convenient_service/service/plugins/displays_result/error/middleware.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module ConvenientService
+  module Service
+    module Plugins
+      module DisplaysResult
+        module Error
+          class Middleware < Core::MethodChainMiddleware
+            def next(**kwargs)
+              puts ::Paint[kwargs[:message].to_s, :red, :bold] if display?(entity)
```

Comment:
Please, implement a "require" method for your dependencies.

Thanks.

Here is an example:

https://github.com/marian13/convenient_service/blob/main/lib/convenient_service/dependencies.rb#L72